src/strands/experimental/bidi/models/gemini_live.py

```
@@ -169,41 +156,20 @@ async def _send_message_history(self, messages: Messages) -> None:
                 # "assistant" role from Messages format maps to "model" in Gemini
                 role = "model" if message["role"] == "assistant" else message["role"]
                 content = genai_types.Content(role=role, parts=content_parts)
-                if self.live_session:
-                    await self.live_session.send_client_content(turns=content)
+                await self._live_session.send_client_content(turns=content)
 
     async def receive(self) -> AsyncIterable[BidiOutputEvent]:  # type: ignore
         """Receive Gemini Live API events and convert to provider-agnostic format."""
-        # Emit connection start event
```

Changes in this method include:
* Raise runtime error if model not "started".
* Remove try/except. Across all model providers, we are going to let exceptions propagate and handle logging centrally in the agent loop.
* Remove yield in finally. In general, it is not appropriate to yield in a finally inside a generator. This can lead to hanging when exceptions are raised.